src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,260 @@
+        DukeFileUtils.loadSaveData(list);
+        printList(list);
+        Scanner in = new Scanner(System.in);
+
+
+        while (true) {
+            userInput = in.nextLine().trim();
+            String userInputLowerCase = userInput.toLowerCase();
+
+            try {
+                if (userInput.equalsIgnoreCase("bye")) {
+                    System.out.println(GOODBYE_MESSAGE);
+                    DukeFileUtils.saveToFile(list);
+                    break;
+                } else if (userInput.equalsIgnoreCase("list")) {
+                    printList(list);
+                } else if (userInputLowerCase.startsWith("done")) {
+                    markTasksAsDone(userInput, list);
+                    DukeFileUtils.saveToFile(list);
+                } else if (userInputLowerCase.startsWith("todo")) {
+                    String description = extractDescription(userInput);
+                    addTodo(description, list);
+                    DukeFileUtils.saveToFile(list);
+                } else if (userInputLowerCase.startsWith("deadline")) {
+                    String description = extractDescription(userInput);
+                    addDeadline(description, list);
+                    DukeFileUtils.saveToFile(list);
+                } else if (userInputLowerCase.startsWith("event")) {
+                    String description = extractDescription(userInput);
+                    addEvent(description, list);
+                } else if (userInputLowerCase.startsWith("delete")) {
+                    deleteTask(userInputLowerCase, list);
+                    DukeFileUtils.saveToFile(list);
+                } else {
+                    System.out.print(HORIZONTAL_LINE + HELP_MESSAGE + HORIZONTAL_LINE);
+                }
+            } catch (DukeException | IndexOutOfBoundsException e) {
+                System.out.println(e.getMessage());
+            }
+        }
+    }
+
+    private static void addTodo(String description, List<Task> taskList) {
+        Todo newTodo = new Todo(description, TaskType.TODO);
+        taskList.add(newTodo);
+        System.out.print(HORIZONTAL_LINE + "Got it! I've added this task:\n" +
+                newTodo + "\n" + HORIZONTAL_LINE);
+    }
+
+    private static void addEvent(String input, List<Task> taskList) throws DukeException {
+        String[] separated = splitDescriptionFromTiming(TaskType.EVENT, input);
+        if (separated.length == 1) {
+            throw new DukeException("Give me a timing for the event too man come on...");
+        }
+        String description = separated[0];
+        String time = separated[1];
+        Event newEvent = new Event(description, TaskType.EVENT, time);
+        taskList.add(newEvent);
+
+        System.out.print(HORIZONTAL_LINE + "Got it! I've added this task:\n" +
+                newEvent + "\n" + HORIZONTAL_LINE);
+    }
+
+    private static void addDeadline(String input, List<Task> taskList) throws DukeException {
```

Comment:
I realised that most of your methods in duke is static. Why not make them non-static as these methods will not be used by other classes?